### ISSUE TITLE
[Feature Request] - Koboldcpp probability

### ISSUE THREAD

**Issue by kalomaze** [lmg-anon/mikupad#1]
Being able to see probability on the kobold API backend would be a nice addition

**Comment by lmg-anon** [comment]
This depends on https://github.com/LostRuins/koboldcpp/issues/469. Implementing it on koboldcpp should be relatively straightforward, as it's a fork of llama.cpp. I'll likely attempt this later.

**Comment by jojorne** [comment]
I agree, I wrote my 2 cents [there](https://github.com/LostRuins/koboldcpp/issues/469). 😅
